### DISCUSSION TITLE
Reward Distribution December 2020

### DISCUSSION THREAD

**Discussion by ngkong** [Tadpole-finance/dev-rewards-program#3] [General]
Hi fellow developers,

This week we will distribute the Developer Rewards for the period of December 2020. You can see the calculation of rewards in the table below. For the usernames listed please include your ETH address in this discussion room as soon as possible.

As of January 2021, we have slightly changed the procedure for calculating the score. If previously we received Pull Requests in the form of typo and grammar corrections or text, then from the next period PR in the form of text corrections is no longer the subject of reward distribution. This change aims to increase developer participation in developing the code base of Tadpole Finance, instead of just improving the text.

Regardless, if you encounter text errors within the Tadpole Finance ecosystem, feel free to correct them and send the PR to the respective repositories. Although there are no rewards for this fix, your efforts will help the public to get to know Tadpole Finance with more precision. Which indirectly increases the credibility of the Tadpole Finance project.

Thank you for your participation in developing Tadpole Finance!

Reward Table:
![Screenshot from 2021-01-07 14-39-35](https://user-images.githubusercontent.com/22634237/103866427-5b5f0f00-50f8-11eb-9da8-2dc43a91cade.png)

Please update your ETH addresses here:
@KaptenAkro
@prayogih
@rajacebong
@stevangoldduck
@syadluq

**Comment by KaptenAkro** [comment]
![Screenshot_20210107-160721](https://user-images.githubusercontent.com/73269185/103867561-86e6f700-5102-11eb-9d44-03f35f01c41e.jpg)

0xEB50F847D430A0B26Be4EF66cf517b92631C8736

Thanks ngkong.

**Comment by stevangoldduck** [comment]
Thanks for noticing us
Here's my TAD address

0xf8c4e2a31db8cdfbedbd7fd0047bf5c516402cf5

**Comment by syadluq** [comment]
Here's Mine, Syadluq @ngkong , not Syadlug xixixi

0x16DfCeabb08A4f0DEffc28B8d7beA29EE120cab7

**Comment by prayogih** [comment]
i am prayogih @ngkong,
this is my Wallet: 0xE04E0Fec2D4b37bF5860ACbEbA99B72FC2975a04

**Comment by prayogih** [comment]
thanks ngkong,for noticing us

**Comment by stevangoldduck** [comment]
May i know when it will be distributed??

**Reply by syadluq** [reply]
Ask dev in telegram

**Reply by stevangoldduck** [reply]
Thanks syadluq

**Comment by Neetsuki** [comment]
https://github.com/Tadpole-finance/tadpole-finance.github.io/pull/32
Min saya kok gak dimasukin? Sedih min padahal dapet score... :(

**Reply by syadluq** [reply]
https://github.com/Tadpole-finance/tadpole-finance.github.io/pull/31

Your score is on 31, not 32

**Reply by ngkong** [reply]
somehow i missed it! i will revise!

**Comment by rajacebong** [comment]
Here is my address, thank you!

0xBf60D6c1771e45f0B4449aB05dc3907319fb1fA1

**Comment by ngkong** [comment]
Let me recap:

@KaptenAkro 0xEB50F847D430A0B26Be4EF66cf517b92631C8736
@prayogih 0xE04E0Fec2D4b37bF5860ACbEbA99B72FC2975a04
@rajacebong 0xBf60D6c1771e45f0B4449aB05dc3907319fb1fA1
@stevangoldduck 0xf8c4e2a31db8cdfbedbd7fd0047bf5c516402cf5
@syadluq 0x16DfCeabb08A4f0DEffc28B8d7beA29EE120cab7

**Reply by syadluq** [reply]
Mine Correct, Thanks!

**Reply by stevangoldduck** [reply]
![image](https://user-images.githubusercontent.com/36699633/104007088-f8de3f80-51d9-11eb-9a62-98083a1e6ed3.png)
Correct

**Comment by ngkong** [comment]
Sorry to @Neetsuki I missed your PR. The following is the revised reward table:

![photo_2021-01-08_17-18-45](https://user-images.githubusercontent.com/22634237/104003934-b31f7800-51d5-11eb-9148-9e7a2510a228.jpg)

@Neetsuki please provide your Ethereum address in this discussion board.

Thank you!

**Reply by Neetsuki** [reply]
Thank you min... :D
This is my eth address : 0x9a6Ab0c145d96069C63AB499B8aD33a7eE46625e

**Comment by ngkong** [comment]
Final reward calculation:

![photo_2021-01-08_17-31-27](https://user-images.githubusercontent.com/22634237/104005435-aef45a00-51d7-11eb-8c06-6824a63cb87e.jpg)

**Reply by syadluq** [reply]
Sorry but why does pull 31 owned by 2 people?

**Reply by syadluq** [reply]
@KaptenAkro should be 2 Points and @Neetsuki  1 point, CMIIW

**Comment by ngkong** [comment]
I revise again because #31 is duplicated

![photo_2021-01-08_17-45-22](https://user-images.githubusercontent.com/22634237/104006824-9dac4d00-51d9-11eb-881d-017f7004700d.jpg)

**Comment by ngkong** [comment]
rewards sent. thank you all for your contribution!

**Reply by Neetsuki** [reply]
Thank you sir... <3

**Comment by KaptenAkro** [comment]
Awesome. Thank you ngkong. Glad the obstacles already cleared up now. Good
luck with the project.
